From: =?utf-8?q?Guido_G=C3=BCnther?= <agx@sigxcpu.org>
Date: Sat, 29 Mar 2014 22:54:44 +0100
Subject: Allow to override calypso's config dir

This can be useful when running automatic tests via e.g. autopkgtest
where we want to have all files in a temporary directory.
---
 calypso/config.py | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/calypso/config.py b/calypso/config.py
index dc17eec..59cbd85 100644
--- a/calypso/config.py
+++ b/calypso/config.py
@@ -72,6 +72,9 @@ for section, values in INITIAL_CONFIG.items():
 
 _CONFIG_PARSER.read("/etc/calypso/config")
 _CONFIG_PARSER.read(os.path.expanduser("~/.config/calypso/config"))
+cfg = os.getenv("CALYPSO_CONFIG")
+if cfg:
+    _CONFIG_PARSER.read(cfg)
 
 # Wrap config module into ConfigParser instance
 sys.modules[__name__] = _CONFIG_PARSER
